### ISSUE TITLE
population weights increase sample size and make everything significant

### ISSUE THREAD

**Issue by albrightb** [runehaubo/lmerTestR#57]
I am using the NHATS which has population weighs that allow national Medicare population estimates, similar to  many US federal nationally-representative datasets. 
When I use my popweight in my multilevel longitudinal logistic model in lme4 and use weights=“popweight” my results make everything significant. This was exciting, but felt wrong. When I mean-centered the weight popweight/mean(popweight), fewer predictors are significant, which tells me that the p-value testing is not basing significance on the original sample size but rather on the weighted (inflated) sample size.

**Comment by runehaubo** [comment]
Yes, the `weights` argument refers to frequency weights, not sampling weights. Giving an observation a weight of 2 is the same as including the observation twice. In any case this has to do with lme4, not lmerTest.
